HousingRepairsOnlineApi/Startup.cs

```diff
@@ -40,6 +42,21 @@ public void ConfigureServices(IServiceCollection services)
                     new EnvironmentVariableRepairTypeSorConfigurationProvider(RepairType.Communal),
                 });
 
+            var sendTenantNotification = new TenantNotificationConfigurationProvider(EnvironmentVariableHelper.GetEnvironmentVariable("TENANT_CONFIRMATION_SMS_NOTIFY_TEMPLATE_ID"),
```

Comment:
The instantiation of each notification configuration provider within the startup _could_ causing longer startup times (and increased memory usage due to always being present and not managed through Dependancy Injection lifetimes). Was this intentional?

Reply:
Yes, I couldn't see another way of doing this, and seemed to fit in with the way we add the SOR engines, which are  created in startup: 

`services.AddSorEngines(
                new[]
                {
                    new EnvironmentVariableRepairTypeSorConfigurationProvider(RepairType.Tenant),
                    new EnvironmentVariableRepairTypeSorConfigurationProvider(RepairType.Communal),
                });`